src/Cdn/Cdn.Autorest/README.md

```diff
@@ -304,4 +484,24 @@ directive:
     where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/endpoints/{endpointName}"].delete
     transform: >-
       $["x-ms-long-running-operation-options"] = {"final-state-via": "azure-async-operation"}
+
+  # announce upcoming MI-related breaking changes
+  - where:
+      parameter-name: IdentityType
+    set:
+      breaking-change:
```

Comment:
Are you planning to roll out breaking changes in this Ignite or just the pre-announcement? I see these information mismatches. We will release Az 13 during Ignite. However, the pre-announcement should be released 2 or 3 sprints before the major release. It is not allowed to roll out breaking changes directly without announcement.